### ISSUE TITLE
Docker CI: should not display `unknown` as commit SHA and commit date

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch#2028]
Since v0.25.0 and the [introduction of the new CI](https://github.com/meilisearch/MeiliSearch/pull/1897), MeiliSearch running on Docker does not display the commit sha and the commit date:

<img width="851" alt="Capture d’écran 2021-12-20 à 17 28 37" src="https://user-images.githubusercontent.com/20380692/146800363-3ae54dc3-c623-453b-978f-2461d927832d.png">

When building locally, I don't have any commit sha and date issue, they are well displayed at MeiliSearch launch.

**Comment by MarinPostma** [comment]
@curquiza, after giving it a quick glance, it seems that [this PR](https://github.com/meilisearch/MeiliSearch/pull/1658) removes the use of env variables for building the image, in favor of importing the `.git` directory. But it seems that we you are still expecting to find these variables in the dockerfile [here](https://github.com/meilisearch/MeiliSearch/blob/release-v0.25.0/Dockerfile#L28-L30), and I see nowhere where the `.git` is actually copied. Not sure how all of that workds, but it may be worth to take a look at that.

**Comment by amab8901** [comment]
I'm unable to reproduce the error for the [latest version of meilisearch](https://hub.docker.com/layers/getmeili/meilisearch/v0.30.0/images/sha256-56d263069d9308a1d75a750f90c3a935b97a37dd888344220ce89f1823f754de?context=explore) on dockerhub. 

<details>
<summary>$ sudo docker run --rm -e MEILI_NO_ANALYTICS=true -p 770:770 getmeili/meilisearch:v0.30.0 ./meilisearch --master-</summary>
```
key=masterKey
Unable to find image 'getmeili/meilisearch:v0.30.0' locally
v0.30.0: Pulling from getmeili/meilisearch
ca7dd9ec2225: Pull complete 
3628dd4a8988: Pull complete 
e5d3cc55b1bb: Pull complete 
b0545ee3798f: Pull complete 
d9faa5b12225: Pull complete 
Digest: sha256:364782ac9f870eefb18259b6c0f4652dcb7c8d375233b6e358a51047a9fc4272
Status: Downloaded newer image for getmeili/meilisearch:v0.30.0
[FATAL tini (7)] exec ./meilisearch failed: No such file or directory

```

</details>

**Comment by curquiza** [comment]
Hello @amab8901, yes we know the bug still exists, @brunoocasali is working on a fix in this PR #2550 😇 
Thanks for letting us know
